xyz-iinuwa-credential-manager-portal-gtk/src/view_model/gtk/mod.rs

```diff
@@ -258,7 +258,11 @@ impl ViewModel {
                 .orientation(gtk::Orientation::Horizontal)
                 .build();
             let icon = gtk::Image::builder().icon_name(icon_name).build();
-            let label = gtk::Label::builder().label(credential.name()).build();
+            let mut display_label = credential.name().to_string();
+            if let Some(username) = credential.username() {
+                display_label += &format!(" ({username})");
```

Comment:
Not sure if we're tracking localisation yet, but we'll need to localise this as a whole string with two placeholders rather than appending.

Reply:
Yeah we're not doing localization since the UI code is meant just as a reference implementation. Eventually, the UI should be implemented by xdg-desktop-portal-gnome itself, or something, which would have proper libadwaita theming and do localization properly.

We can create an issue though to review if this UI code becomes more widespread.